src/Controller/UI/DatasetReviewController.php

```diff
@@ -270,11 +254,8 @@ private function addToFlashDisplayQueue(Request $request, string $udi, string $n
     /**
      * Make the submission form and return it.
      *
-     * @param string            $udi               The UDI entered by the user.
-     * @param Dataset           $dataset           The Dataset.
-     * @param DatasetSubmission $datasetSubmission The Dataset Submission.
-     *
-     * @return Response A Response instance.
+     * @param string            $udi               the UDI entered by the user
+
      */
     protected function makeSubmissionForm(string $udi, Dataset $dataset = null, DatasetSubmission $datasetSubmission = null)
```

Comment:
make private? also, missing return type

Reply:
Yes can, but didn't do this. But in the future you should do that.